src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +344,22 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private void checkForNonASCII(String documentationText, String location) {
+        if (!CharMatcher.ASCII.matchesAllOf(documentationText)) {
+            throw new AssertionError("Non-ASCII character used in documentation of "+callerOptions.getClass().getSimpleName()+" ("+location+"). Only ASCII characters are allowed.");
+        }
+    }
+
+    private String convertFromHtml(final String textToConvert) {
+
+        String regex1 = "<br />|<p>|</p>|<table>|</table >|<a href='(.*)'>(.*)</a>|<h4>|<pre>|</pre>|<ul>|</ul>|</li>|</tr>|<hr />";
+        String regex2 = "</h4>|<tbody>|</tbody>|<tr>|<th>";
+        String regex3 = "<li>";
+        String regex4 = "</th>";
+
```

Comment:
Could you search for any unmatched html tags, in case someone adds one not supported here to the docs?   `regex=(<[^<]+>)`  when the others don't match.